app/presenters/document/field_group_presenter.rb

```diff
@@ -133,7 +133,7 @@ def section_field_subsection(section)
           if section[:entity_extra].present?
             possible_entities = entities(section[:entity_name], section[:entity_proxy_field])
             salient_entity = possible_entities.detect do |entity|
-              entity_label(entity, section[:entity_name]).any? { |label| label == val }
+              [entity_label(entity, section[:entity_name])].any? { |label| label == val }
```

Comment:
entity_label *may* return a string but will more often return an array, so this needs to be `[entity_label(entity, section[:entity_name])].flatten.any? { |label| label == val }`